data/schema.go

```diff
@@ -321,4 +321,8 @@ const (
 
 	PointTypeTimeSync  = "timeSync"
 	PointTypeConnected = "connected"
+
+	NodeTypeNetworkManager       = "networkManager"
+	NodeTypeNetworkManagerDevice = "networkManagerDevice"
+	NodeTypeNetworkManagerConn   = "networkManagerConn"
```

Comment:
Should we also add the point types?

Reply:
we probably should at some point to complete schema and documentation